src/init.cpp

```diff
@@ -1467,7 +1467,15 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     if (mempool_opts.max_size_bytes < 0 || mempool_opts.max_size_bytes < descendant_limit_bytes) {
         return InitError(strprintf(_("-maxmempool must be at least %d MB"), std::ceil(descendant_limit_bytes / 1'000'000.0)));
     }
-    LogPrintf("* Using %.1f MiB for in-memory UTXO set (plus up to %.1f MiB of unused mempool space)\n", cache_sizes.coins * (1.0 / 1024 / 1024), mempool_opts.max_size_bytes * (1.0 / 1024 / 1024));
+    // If we are running in blocksonly mode don't allocate the default 300MB of space for mempool
+    // unless the user specifies -maxmempool, as otherwise it will leak into dbcache allowance.
```

Comment:
In quite a few places we refer to mempool cache being added to dbcache, but it's added specifically to the coins/UTXO cache so I think our language should reflect that consistently? (here and in other places)

```suggestion
    // unless the user specifies -maxmempool, as otherwise it will leak into UTXO cache allowance.
```

Reply:
Agreed and changed.